views/pulsar/v2/helpers/form.html.twig

```diff
@@ -2107,3 +2107,87 @@ data-*        | string | Data attributes, eg: `'data-foo': 'bar'`
 
 {% endspaceless %}
 {% endmacro %}
+
+{% macro repeatable_start(options) %}
+{% import '@pulsar/pulsar/v2/helpers/elem.html.twig' as elem %}
+{% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
+{% import _self as form %}
+{% set headingLength = options.headings|length %}
+{% spaceless %}
+    <div
+        class="repeater"
+        data-repeater-add-new-group-text="{{ options['add-new-group-text']|default('Add') }}"
+        data-repeater-add-another-group-text="{{ options['add-another-group-text']|default('Add another') }}"
+        data-repeater-max-entries="{{ options['max-entries']|default(false) }}"
+    >
+        <div class="repeater__saved-data" style="display: none;"></div>
+        <div class="form__group">
+            {{
+                elem.label({
+                    'label': options.label,
+                    'required': options.required|default(false)
+                })
+            }}
+            <div class="controls">
+                <div class="table-container">
```

Comment:
This wrapper is added by [PulsarUIComponent](https://github.com/jadu/pulsar/blob/develop/js/PulsarUIComponent.js#L48) so could be omitted here if preferred

Reply:
Nice! Removed